### DISCUSSION TITLE
Can we customize overview API page ?

### DISCUSSION THREAD

**Discussion by t-prod** [dedoc/scramble#38] [Q&A] [answered]
Hi,

Thanks for this awesome package. It's a real pleasure to use and it is out of the box.
Is there a way to customize the overview API page ? literally the homepage when we navigate to docs/api ?

Thanks for your help

**Comment by romalytvynenko** [comment]
Hey @t-prod 

It will be possible, I'm on it. Please follow the issue https://github.com/dedoc/scramble/issues/23.

P.S.: It is possible to add description to Open API document which will be rendered on the home (`/docs/api`) page. However you can render only some markdown there.

**Comment by romalytvynenko** [comment, marked as answer]
Hey @t-prod 

Starting from `0.4.0` you can do that!

Simply publish Scramble's config and update `scramble.info.description` value. It will be used as a content on the home page.

Here is docs: https://scramble.dedoc.co/usage/info

🙌
